components/admin_console/license_settings/__snapshots__/license_settings.test.jsx.snap

```diff
@@ -213,7 +213,7 @@ exports[`components/admin_console/license_settings/LicenseSettings load screen w
               restartError={null}
               restarting={false}
               upgradeError={null}
-              upgradingPercentage={0}
+              upgradingPercentage={42}
```

Comment:
Where does 42 come from?

Reply:
automatically set by the upgrade snapshot functionality from tests. Same file, line 136

Reply:
Ah gotcha, didn't notice it was the `.snap` 🤦‍♀️